### ISSUE TITLE
Snapshot publish failed

### ISSUE THREAD

**Issue by armanbilge** [typelevel/scalacheck#881]
And therefore CI publishing can be assumed broken, sigh.

https://github.com/typelevel/scalacheck/runs/5666274206

**Comment by rossabaker** [comment]
There are four source files in `src` that also appear in `core/$PLATFORM/src`.  Perhaps a bad merge somewhere?

**Comment by armanbilge** [comment]
Oh, good catch. Not sure if it's related but one more thing to fix!

**Comment by armanbilge** [comment]
Looking more carefully at the logs, that does seem like it could be the issue.

**Comment by armanbilge** [comment]
Yeah, you are spot on :)

Its because those sources belonged to `root` and the publish job is the only one that runs on `root`. Every other job runs on either `rootJVM`, `rootJS`, or `rootNative` which wouldn't see those misplaced srcs.

**Comment by armanbilge** [comment]
Bah, still broken. I forgot to kill the Native/3 build.

**Comment by SethTisue** [comment]
#879 is merged, so I think this is the next thing that needs resolution before we can publish?

**Comment by armanbilge** [comment]
Gah, still broken! I think the secrets on this repository have not been removed so it's used the old credentials. @SethTisue do you admin access on this repo to remove them?

**Comment by armanbilge** [comment]
https://github.com/typelevel/scalacheck/pull/868 can be closed, it was superseded by https://github.com/typelevel/scalacheck/pull/879.

**Comment by SethTisue** [comment]
> @SethTisue do you have admin access on this repo to remove them?

I don't

**Comment by armanbilge** [comment]
Oh, I was being foolish. The Typelevel sonatype credentials actually won't help us at all. We need someone's credentials for publishing under `org.scalacheck`.

**Comment by SethTisue** [comment]
> We need someone's credentials for publishing under org.scalacheck

@djspiewak ? @ChristopherDavenport ?

**Comment by djspiewak** [comment]
I have them but I don't have my laptop at present. Can work on it this weekend if Chris doesn't get to things first.

**Comment by armanbilge** [comment]
Whoever installs credentials should add the following secrets:
- `SONATYPE_USERNAME` and `SONATYPE_PASSWORD`
- `SONATYPE_CREDENTIAL_HOST`: `oss.sonatype.org`

`PGP_SECRET` and `PGP_PASSPHRASE` should not be present. These will be inherited from the org-level secrets.

**Comment by armanbilge** [comment]
Ok, new plan: I opened this issue to migrate scalacheck to s01 and grant the typelevel-bot publishing rights.
https://issues.sonatype.org/browse/OSSRH-79416

@djspiewak can you confirm there please? Thanks!

**Comment by armanbilge** [comment]
Update: Chris has kindly confirmed on that issue. So now we wait for sonatype to process the changes, and then everything should start magically working 🤞

**Comment by SethTisue** [comment]
restarted

**Comment by armanbilge** [comment]
Thank you!! Everything seems to be working :)
https://s01.oss.sonatype.org/content/repositories/snapshots/org/scalacheck/scalacheck_native0.4_3/1.16-0ac8005-SNAPSHOT/
